website/content/docs/sync/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+page_title: Secrets Sync
+description: Secrets sync allows you to safely sync Vault-managed secrets with external destinations.
+---
+
+# Secrets Sync
+
+Secrets sync enables users to manage their secrets with Vault while syncing a subset of them into
+external secret store destinations. Secrets that will be synced must be individually associated
+with each desired destination. This means is associating a secret named `my-secret` with a
+destinated named `my-aws` will not automatically sync to second destination named `my-gcp`.
+
+<Note>
+	Unsyncing a secret either by deleting the secret from Vault or from deleting its association
+	is a hard-delete option. This means it is immediately removed from the external system and bypasses
+	any soft-deletion states or recovery mechanisms.
+</Note>
+
+## Destinations
+
+Secrets can be synced into various external stores, called destinations. Destinations are configured
+separately for each type of external system is being used. The supported destinations are listed:
+* [AWS Secrets Manager](/vault/docs/sync/awssm)
+* [Azure Key Vault](/vault/docs/sync/azurekv)
+* [GCP Secret Manager](/vault/docs/sync/gcpsm)
+* [GitHub Repository Actions](/vault/docs/sync/github)
+* [Vercel Projects](/vault/docs/sync/vercelproject)
+
+## Assocations
+
+Choosing the secrets you want synced to a given destination is done by creating links between them,
+called associations. Once a destination has been successfully configured, you can begin making the
+associations to it by providing the secret's name and its mount. Each secret's association keeps
+track of its current sync status, the timestamp of its last status change, and the error code of
+the last sync or unsync operation, if it failed.
+
+
+## Sync Statuses
+
+There are several sync statuses which relay information about the outcome of the last sync
+operation to have occurred on that secret. The status information is stored inside each
+association object returned by the endpoint and, upon failure, includes an error code explaining why.
```

Comment:
`returned by the endpoint`

How can a user know what this endpoint is and how to query it?

Should we include a small list of examples with their resulting statuses to teach users what they look like and what might be the cause of some of the errors?